project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -39,20 +46,22 @@
 
     /** In this radius around "LOCATION" places will be searched. */
     private static final int SEARCH_RADIUS = 5000;
-    // TODO(M1): check at least 10 results, and if less extend radius
 
     /** The entry point for a Google GEO API request. */
     private static final GeoApiContext CONTEXT = new GeoApiContext.Builder()
         .apiKey(System.getenv("API_KEY"))
         .build();
 
+    /** A mapping between cuisines and text search words. */
+    private static final Map<String, String[]> CUISINE_TO_SEARCH_WORDS_MAP = getCuisinesMap();
```

Comment:
_MAP is redundant, "CUISINE_TO_SEARCH_WORDS" is descriptive enough along with the type (as it's a map).

Reply:
Still has _MAP suffix on my end.

Reply:
Wierd, for me it apears here on github without the _MAP suffix.
Maybe not refreshed?